client/material/select/select.tsx

```diff
@@ -244,25 +245,27 @@ export const Select = React.forwardRef<SelectRef, SelectProps>(
       [anchorRef],
     )
 
-    const [displayValue, selectedIndex] = useMemo(() => {
-      if (value === undefined) {
-        return [undefined, undefined]
-      }
+    const [displayValue, options] = useMemo(() => {
+      let displayText: string | undefined
+      const options = React.Children.map(children, (child, index) => {
+        if (!isSelectedMenuItem(child)) return child
```

Comment:
What exactly is the point of `SelectedMenuItem`? Is it that the item always appears as selected? Is it that item *could* be selected? I'm really not sure because the naming implies the former, but this code implies the latter